src/components/CalendarDay.jsx

```diff
@@ -48,6 +55,9 @@ export default class CalendarDay extends React.Component {
         onClick={e => this.onDayClick(day, e)}
       >
         <span className="CalendarDay__day">{day.format('D')}</span>
+        {details && (
+          <span className="CalendarDay__details">{details}</span>
```

Comment:
@majapw if you look through the individual commits, I added this in 0531054 so that the `details` span wouldn't render if it was empty; maybe that wasn't needed?